Modules/_xxsubinterpretersmodule.c

```diff
@@ -2323,6 +2353,74 @@ PyDoc_STRVAR(channel_list_all_doc,
 \n\
 Return the list of all IDs for active channels.");
 
+
+static PyObject *
+channel_list_interpreters(PyObject *self, PyObject *args, PyObject *kwds)
+{
+    static char *kwlist[] = {"cid", "send", "recv", NULL};
+    int64_t cid;
+    _PyChannelState *chan;
+    int send = 0;           /* Send end? */
+    int recv = 0;           /* Receive end? */
+    int64_t *ids = NULL;    /* Array of interpreter IDs */
+    int64_t count = 0;      /* Number of interpreters to return */
+    PyObject *ret = NULL;   /* Python list of interpreter IDs */
+
+    if (!PyArg_ParseTupleAndKeywords(
+            args, kwds, "O&|$pp:channel_list_interpreters",
+            kwlist, channel_id_converter, &cid, &send, &recv)) {
+        return NULL;
+    }
+
+    if ((send && recv) || (!send && !recv)) {
+        PyErr_SetString(PyExc_ValueError,
+                        "Specify exactly one of send or recv");
```

Comment:
If it can only be one then why not drop either `send` or `recv`.  Then the signature is simpler and you can drop this check.

FWIW, I'd keep just `recv` (though you chose `send`) and make the default value `True`.  That's consistent with how I've tried to favor `recv` elsewhere when both ends are involved.  (In my mind, conceptually `recv` is the more fundamental operation of the two.)